apps/network-alerts/js/attention/attention.js

```diff
@@ -31,26 +30,24 @@ function sendNotification() {
   }
 
   return new Promise(function(resolve, reject) {
```

Comment:
Ideally here we can return the promise chain because the mozApps.getSelf() behave like a promise:
```
return navigator.mozApps.getSelf().then((app) => {
  ...
  return NotificationHelper.send(...);
});
```
Just a suggestion, not a must have in this patch.